opentelemetry-sdk/src/opentelemetry/sdk/logs/export/__init__.py

```diff
@@ -0,0 +1,53 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import abc
+import enum
+from typing import Sequence
+
+from opentelemetry.sdk.logs import LogData
+
+
+class LogExportResult(enum.Enum):
```

Comment:
Is this part of the spec?

Reply:
No, I think this will be eventually added to spec. Even I can send PR to get this ExportResult included in spec if you want.